app/open_deep_research.py

```
@@ -1,4 +1,5 @@
 import uuid
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)